compiler/tflchef/tflite/src/Op/UnidirectionalSequenceLSTM.cpp

```diff
@@ -30,12 +30,6 @@ void TFliteOpUnidirectionalSequenceLSTM::filler(const tflite::Operator *op, TFli
 
   for (int32_t i = 0; i < inputs.size(); i++)
   {
-    // Except for Input 0, 17 and 18.
-    // Each Input mean Input[0](=Input Tensor), Input[17](=OutputState Tensor) and
-    // Input[18](=CellState Tensor).
-    // This could be updated from previous input or User Given data, so This could not be Const
-    if (i == 0 || i == 17 || i == 18)
-      continue;
```

Comment:
As comment said, this could not be CONST. Have it changed?

Reply:
This might as well be merged as we're not sure about this. It could be changed after testing with real model.

Reply:
@mhs4670go I have a little question. is there a way to create variable tensor using tflchef?